meta-lhg/recipes-browser/chromium/chromium-wayland_%.bbappend

```diff
@@ -1,4 +1,6 @@
 PACKAGECONFIG_append = " ${@bb.utils.contains('MACHINE_FEATURES', 'optee', 'use-ocdm', '', d)}  proprietary-codecs"
+PACKAGECONFIG[use-ocdm] = ",,,"
```

Comment:
Just 
PACKAGECONFIG[use-ocdm] = ""
is enough

Reply:
I updated the code base on this comment.